web/src/compose_closed_ui.ts

```diff
@@ -137,14 +137,40 @@ export function update_buttons_for_private(): void {
     update_buttons(disable_reply);
 }
 
+function get_current_dm_recipient_ids_string(): string {
+    const selected_message = message_lists.current?.selected_message();
+    const recipient_id = selected_message?.display_recipient;
+
+    if (Array.isArray(recipient_id)) {
+        return recipient_id.map((recipient: {id: number}) => recipient.id).join(",");
+    }
+    return "";
+}
+
 export function update_buttons_for_stream_views(): void {
     $("#new_conversation_button").attr("data-conversation-type", "stream");
     update_buttons(should_disable_compose_reply_button_for_stream());
 }
 
 export function update_buttons_for_non_specific_views(): void {
-    $("#new_conversation_button").attr("data-conversation-type", "non-specific");
-    update_buttons(should_disable_compose_reply_button_for_stream());
+    let conversation_type = "non-specific";
```

Comment:
This is never used right now. I think we can use `non-specific` we are not sure about the type of message.